### DISCUSSION TITLE
Multipart 매핑문제 해결하기

### DISCUSSION THREAD

**Discussion by amaran-th** [BDD-CLUB/challenge-every-week-posting#184] [1. 개발 관련 글 모음]
관리자 페이지를 만드니 한 프로젝트에 안드로이드 백엔드 프론트엔드가 모두 공존하게 되었네요ㅋㅋㅋ
프론트 작업 하면서 multipart 매핑하는 데 문제를 겪었는데, 그 과정을 정리해보았습니다...
사실 별 내용은 없습니다. 문제 원인을 끝까지 못찾아서리...
[게시글 링크](https://amaran-th.github.io/%EC%A3%BC%EC%A0%80%EB%A6%AC%EC%A3%BC%EC%A0%80%EB%A6%AC/[Kerdy]%20Multipart%20%EC%9A%94%EC%B2%AD%20%EB%8D%B0%EC%9D%B4%ED%84%B0%20%EB%A7%A4%ED%95%91%20%EB%AC%B8%EC%A0%9C%20%ED%95%B4%EA%B2%B0/)

**Comment by pnudpwls1103** [comment]
변화를 줘도 똑같은 에러가 뜨면 매우 답답하죠.. 고생하셨습니다!!
